extensions/answer_summarizers/models_test.py

```diff
@@ -16,15 +16,27 @@
 
 """Test calculations to get interaction answer views."""
 
+import itertools
+
+from extensions.answer_summarizers import models as answer_models
 from core.domain import calculation_registry
 from core.domain import exp_domain
 from core.tests import test_utils
 
 
-class InteractionAnswerSummaryCalculationUnitTests(test_utils.GenericTestBase):
-    """Tests for answer summary calculations."""
+class BaseCalculationUnitTest(test_utils.GenericTestBase):
```

Comment:
Suggest BaseCalculationUnitTests -- it has one now, it may have more later.

Reply:
Done.